// angular2-materialize/__ivy_ngcc__/dist/materialize-module.js
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { MaterializeDirective } from './materialize-directive';
import * as ɵngcc0 from '@angular/core';
var MaterializeModule = (function () {
    function MaterializeModule() {
    }
    /**
     * @return {?}
     */
    MaterializeModule.forRoot = function () {
        return {
            ngModule: MaterializeModule
        };
    };
MaterializeModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: MaterializeModule });
MaterializeModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function MaterializeModule_Factory(t) { return new (t || MaterializeModule)(); }, imports: [[
            CommonModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(MaterializeModule, { declarations: function () { return [MaterializeDirective]; }, imports: function () { return [CommonModule]; }, exports: function () { return [MaterializeDirective]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(MaterializeModule, [{
        type: NgModule,
        args: [{
                declarations: [
                    MaterializeDirective
                ],
                imports: [
                    CommonModule
                ],
                exports: [
                    MaterializeDirective
                ]
            }]
    }], function () { return []; }, null); })();
    return MaterializeModule;
}());
export { MaterializeModule };
/**
 * @nocollapse
 */
MaterializeModule.ctorParameters = function () { return []; };
function MaterializeModule_tsickle_Closure_declarations() {
    /** @type {?} */
    MaterializeModule.decorators;
    /**
     * @nocollapse
     * @type {?}
     */
    MaterializeModule.ctorParameters;
}
